client/components/ActivityCard.jsx

```diff
@@ -8,29 +8,39 @@ class ActivityCard extends React.Component {
   clickHandler = e => {
     const userId = this.props.user_id
     this.props.addActivity(userId, {
-        activityId: e.target.id,
-        rating: e.target.getAttribute('value'),
-        log: e.target.getAttribute('name')
+      activityId: e.target.id,
+      rating: e.target.getAttribute('value')
     })
   }
 
   renderSmiles = () => {
     // this.props.activity && Number(this.props.activity.rating) === smile.value && { color: 'green' }
     const { card, smiles } = this.props
 
-    return smiles.map((smile, key) => 
+    return smiles.map((smile, key) => (
       <a key={key}>
         <i
-          className={
-            'far ' + `${smile.mood}` + ' fa-3x facesInCss'
-          }
+          className={'far ' + `${smile.mood}` + ' fa-3x facesInCss'}
           value={smile.value}
           id={this.props.act_id}
           name={this.props.name}
           onClick={this.clickHandler}
-          style={card && card.rating === Number(smile.value) ? { color: smile.color } : {}}
+          style={
+            card && card.rating === Number(smile.value)
+              ? { color: smile.color }
+              : {}
```

Comment:
What does the empty object stand for? no colour? (I know it works, just intrigued)